src/pages/ApplyPage.tsx

```diff
@@ -154,6 +154,18 @@ function ApplyPage() {
             <li>Additional team meetings may be arranged by your Project Manager when necessary</li>
             "
           />
+          <Accordion
+            title="How are projects directed?"
```

Comment:
Better title for question: "How do teams pick an idea for their project?"

Reply:
Completed.